pkg/azurestore/azureservice.go

```diff
@@ -309,3 +343,37 @@ func checkForNotFoundError(err error) error {
 	}
 	return err
 }
+
+// simple parse http ranging, no multipart ranges/no if-range/no last-modified, not supported by azure anyway
+func parseHTTPRange(r *http.Request) (*azblob.DownloadStreamOptions, error) {
+	rangeHeader := r.Header.Get("Range")
+	if rangeHeader == "" {
+		// this is totally fine, Range header is not required
+		return nil, nil
+	}
+
+	const prefix = "bytes="
+	if !strings.HasPrefix(rangeHeader, prefix) {
+		return nil, fmt.Errorf("invalid Range header format")
+	}
+
+	rangeParts := strings.Split(strings.TrimPrefix(rangeHeader, prefix), "-")
+	if len(rangeParts) != 2 {
+		return nil, fmt.Errorf("invalid Range header format")
+	}
+
+	offset, err := strconv.ParseInt(rangeParts[0], 10, 64)
+	if err != nil {
+		return nil, fmt.Errorf("invalid offset in Range header")
+	}
+
+	count, err := strconv.ParseInt(rangeParts[1], 10, 64)
```

Comment:
You mentioned that "Incomplete range requests (Range: bytes=100- ) are not supported". From the SDK's documentation, it seems to be possible to let the count be zero and get the entire resource starting at the offset:

https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/storage/azblob@v1.5.0/internal/exported#HTTPRange

> An HTTPRange which has an offset and zero value count indicates from the offset to the resource's end. 

Could you implement support for this?